src/league-of-legends-api/poll-current-game.ts

```diff
@@ -204,9 +204,13 @@ export const stopPollingLoLGame = () => {
   // Stop polling if there is currently a valid polling timer
   if (leagueOfLegendsPollTimer) {
     clearInterval(leagueOfLegendsPollTimer);
-    console.log('No longer in game, polling stopped.');
+    console.log('Stopping polling..');
     cachedEvents = [];
     cachedGame = null;
     leagueOfLegendsPollTimer = null;
   }
 };
+
+export const isPolling = (): boolean => {
+  return leagueOfLegendsPollTimer !== null;
+};
```

Comment:
Super small styling nit but since this function is simple, I think it would look cleaner implicitly returned e.g. `export const isPolling = (): boolean => leagueOfLegendsPollTimer !== null;`

Reply:
Updated